tasks/elasticsearch-RedHat-version-lock.yml

```diff
@@ -2,13 +2,13 @@
 - name: RedHat - install yum-version-lock
   become: yes
   yum:
```

Comment:
Should this then better be `package` to be more generic?

Reply:
Also sensible